### PR TITLE
Adds Squish Emote. Please stop squishing at me so lewdly.

### PR DESCRIPTION
**Pull request by Xantholne**
## About The Pull Request

Adds *squish emote

## Why It's Good For The Game

We have barks, beeps, pings, nyas, chitters, but nothing for slimes in terms of a vocal emote until now.

## Changelog
:cl:
add: You can now *squish
/:cl:

### CODE REVIEW

File: modular_citadel/code/modules/mob/cit_emotes.dm [hunk 1]
```diff
@@ -244,3 +244,20 @@
...
 	var/sound = pick('modular_citadel/sound/voice/bark1.ogg', 'modular_citadel/sound/voice/bark2.ogg')
 	playsound(user, sound, 50, 1, -1)
+
+/datum/emote/living/squish
+	key = "squish"
+	key_third_person = "squishes"
+	message = "squishes!"
+	emote_type = EMOTE_AUDIBLE
+	muzzle_ignore = FALSE
+	restraint_check = false
```
Comment (Ghommie) [inline modular_citadel/code/modules/mob/cit_emotes.dm]:
```suggestion
	restraint_check = FALSE
```

**Review by Ghommie — CHANGES_REQUESTED** [review]
Simple emotes, clothing and trinkets alone don't make the game any more interesting. I wish you would make more meaningful features instead.

**Comment by ghost** [comment]
no more modular citadel pls

**Comment by Ghommie** [comment]
Not this time, sorry.

**Pull request closed without merging**